### ISSUE TITLE
Request form: Org name error focus

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2909]
### Issue description

URL: [Organization name and mailing address | Request a .gov | .gov Registrar (cloud.gov)](https://getgov-staging.app.cloud.gov/request/organization_contact/)
Test ID: 4.F Focus order meaning
Issue: When leaving required form fields blank and continuing to the next page, the page gets refreshed and the focus shifts the beginning of the main content (Previous page). However, when using a screen reader, the first thing that is announced to assistive technology users is the URL and when you press the TAB key, the focus starts at the first interactive element in the main content area. Users may be confused when the focus suddenly shifts without warning as they may not understand why it moved or what action to take next. The field in error is identified and a message is shown visually but no announcement is made to screen readers, hindering them to effectively complete tasks and interact with the website. To fix the issue, the focus should move to the error message at the top of the page or if that is not possible, there must be an audible announcement and focus can either go to the first field in error or remain on the button that triggered the error.

### Acceptance criteria

- [ ] When a user attempts to proceed to the next step of the form and there is an error on this page, the focus goes to the first field in error.

### Links to other issues

- Complete in #3149 

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**
